vendor/engines/c2po/spec/models/credit_card_account_spec.rb

```diff
@@ -37,7 +37,7 @@
   end
 
   it "should be limited to a single facility" do
-    expect(CreditCardAccount.limited_to_single_facility?).to be true
+    expect(CreditCardAccount.single_facility?).to be true
```

Comment:
I wasn't going to comment on this initially, because my preference for `be_xxx` isn't that strong, but can we make the credit_card_account_spec and purchase_order_account spec consistent? Either use `single_facility?) to be true` or `.to be_single_facility`.